gorapass/urls.py

```diff
@@ -5,5 +5,6 @@
 urlpatterns = [
     path('', views.index, name='index'),
     path('populate_stamps_datatable', views.populate_stamps_datatable, name='populate_stamps_datatable'),
-    path('populate_hikes_datatable', views.populate_hikes_datatable, name='populate_hikes_datatable')
+    path('populate_hikes_datatable', views.populate_hikes_datatable, name='populate_hikes_datatable'),
+    path('empty_hikes_datatable', views.empty_hikes_datatable, name='empty_hikes_datatable')
```

Comment:
Let's make a habit of leaving a trailing comma (guilty ✋🏻) so that these types of changes only show the added line, not the deletion and add just for the comma. We can start on this one.

Reply:
👍